documentation/src/main/asciidoc/topics/ref_server_realm_trust.adoc

```diff
@@ -1,18 +1,26 @@
-[id='trust_store_realm-{context}']
+[id='trust-store-realms_{context}']
 = Trust Store Realms
-Trust store realms use keystores that contain certificates, or certificate chains, that clients must present to connect to {brandname} Server.
+{brandname} Server can encrypt traffic using industry standard SSL/TLS protocols.
```

Comment:
I believe the main objective of `Trust Store Realms` is to provide authentication with the client certificate. The "encrypted" traffic is just a side effect and it is not relevant to this section (IMO).

Probably, remove this sentence and the next one and say something like 
`Trust Store Realms use certificates, or certificates chain, stored in keystores that the clients must present to be able to connect to Infinispan server`

Reply:
I've reworded this. Tristan commented earlier and I've tried to combine his content with your suggestion.